src/popup.ts

```diff
@@ -117,7 +117,7 @@ function genNotif (notif: NotifElm): string {
 				<div class="notif-wrap">
 					<img class="notif-img" src="${notif.imgSrc}">
 					<p class="author-note">${notif.authorNote}</p>
-					${notif.content && `<p class="notif-content">${notif.content}</p>`}
```

Comment:
This is... odd. Not sure why this code would cause text overlap. It simply checks if notif.content exists, and adds the content if so. I noticed in the Notification interface that [notif.content is optional](https://github.com/ka-extension/ka-extension-ts/blob/b53a4e7c76fa3d05ad95e4e71837ce420c422328/src/types/data.ts#L92), so this check makes sense to have in case it's not there, and I'm not quite convinced on removing it.

Reply:
I _think_ it's to do with the margin stuff, and since I've made it so that the <p> tag exists whatever the case, the margin is kept.
I'm really new to all this code so I don't quite know what I'm doing yet xD

Reply:
But since these types of notifications have content anyway, I'll probably revert this change.

Reply:
Ah, I think you're correct, the `notif-content` class gives it left padding of 50px, [seen here](https://github.com/ka-extension/ka-extension-ts/blob/b53a4e7c76fa3d05ad95e4e71837ce420c422328/popup/popup.css#L131). Doesn't seem like a good way to pad something TBH, evidenced by this overlap bug, lol.

And hey, no problem, I appreciate you working on stuff!

Reply:
For avatar rewards, there is a different API entry for the content text, so all I had to do is set it to that.

![image](https://user-images.githubusercontent.com/49789044/109923422-ad668000-7cb6-11eb-8ac6-d0f9eae9c6ac.png)

So I've kept the code above that I removed before.